updatesnap/updatesnapyaml.py

```diff
@@ -54,6 +54,12 @@ def get_yaml_file(self, project_url):
                 data = self._github.get_file(project_url, yaml_path)
             except (ValueError, ConnectionError):
                 data = None
+        if not data:
```

Comment:
This gives priority to 'snap/snapcraft.yaml' over 'rockcraft.yaml', while the previous code was the opposite. Which one should have priority?

Reply:
It does not matter which is more preferred, as it is highly discouraged to have both `snapcraft.yaml` and `rockcraft.yaml` in a single repository. Snaps and rocks are meant for different purposes. Since `desktop-snaps` was initially meant only for `snapcraft.yaml`, it should have higher preference

Reply:
Perfect. Thanks!